damnit/backend/db.py

```diff
@@ -349,6 +396,24 @@ def items(self):
         return ItemsView(self.to_dict())
 
 
+# Messages to notify clients about database changes
+
+class MsgKind(Enum):
```

Comment:
I've put the message definition stuff in with the database machinery, because the messages are all describing database changes.

But for now I've left the responsibility for sending messages outside, so you call a database write method and then send the corresponding message. We could tie these closer together, so a call to a method like `db.update_computed_variables()` can also send out notifications. I'm in two minds over this, but it can always be refactored later.